src/Traits/HasTranslations.php

```diff
@@ -65,7 +65,7 @@ public function getTranslations($key): array
     public function attributesToArray()
     {
         $values = array_map(function ($attribute) {
-            return $this->getTranslation($attribute, config('app.locale')) ?: null;
+            return $this->getTranslation($attribute, config('app.locale')) ?: $this->getTranslation($attribute, config('translatable.fallback_locale'));
```

Comment:
Use `config('app.fallback_locale')` instead of `config('translatable.fallback_locale')`